openlr/graph.hpp

```diff
@@ -25,11 +26,11 @@ class Graph
 
   Graph(Index const & index, std::shared_ptr<routing::CarModelFactory> carModelFactory);
 
-  void GetOutgoingEdges(routing::Junction const & junction, EdgeVector & edges) const;
-  void GetIngoingEdges(routing::Junction const & junction, EdgeVector & edges) const;
+  void GetOutgoingEdges(routing::Junction const & junction, EdgeVector & edges);
```

Comment:
Please, add a comment describing what's the difference between these two methods and a couple of methods below.

Reply:
Done.